backend/tests/unit/apps/owasp/models/crp/certificate_test.py

```
@@ -0,0 +1,114 @@
+        """Test __str__ for revoked certificate."""
+        user = User(login="jane_doe")
+        cert = Certificate(github_user=user, tier=TierChoices.LEVEL_2, is_revoked=True)
+
+        assert str(cert) == "jane_doe - LEVEL_2 Certificate (Revoked)"
+
+    @patch("django.db.transaction.Atomic.__enter__", return_value=None)
+    @patch("django.db.transaction.Atomic.__exit__", return_value=None)
+    @patch(f"{MODEL_PATH}.Certificate.objects")
+    @patch(f"{MODEL_PATH}.User.objects")
+    def test_issue_certificate_already_exists(
+        self, mock_user_objects, mock_cert_objects, mock_exit, mock_enter
```

Ruff (0.16.1)

[warning] Unused method argument: `mock_exit`

(ARG002)